docs/user/metal/install_upi.md

```diff
@@ -0,0 +1,107 @@
+# Install: User Provided Infrastructure
+
+The steps for performing a UPI-based install are outlined here. Several [Terraform][terraform-templates] templates are provided to help model your own.
```

Comment:
@abhinavdahiya you think is sensible to extend the above sentence to suggest that the terraform templates are just an example however the user/ operator is free to model their own infrastructure using whatever CM/ orchestration tool they want ? 

i believe by doing so nobody (external to RH) will get confuse, especially with this [info](https://github.com/openshift/openshift-ansible/pull/11338) and so the RH story will flow

my 0.02$